api/v11/api_v11_replication.go

```diff
@@ -0,0 +1,60 @@
+package v11
+
+import (
+	"context"
+	"github.com/dell/goisilon/api"
+)
+
+const (
+	policiesPath = "/platform/11/sync/policies/"
+)
+
+// Policy contains the CloudIQ policy info.
+type Policy struct {
+	Action       string `json:"action,omitempty"`
+	Id           string `json:"id,omitempty"`
+	Name         string `json:"name,omitempty"`
+	Enabled      bool   `json:"enabled,omitempty"`
+	TargetPath   string `json:"target_path,omitempty"`
+	SourcePath   string `json:"source_root_path,omitempty"`
+	TargetHost   string `json:"target_host,omitempty"`
+	JobDelay     int    `json:"job_delay,omitempty"`
+	Schedule     string `json:"schedule,omitempty"`
+}
+
+type Policies struct {
+	Policy []Policy `json:"policies,omitempty"`
+}
+
+// GetPolicyByName returns policy by name
+func GetPolicyByName(
+	ctx context.Context,
+	client api.Client, name string) (policy *Policy, err error) {
+	p := &Policies{}
+	err = client.Get(ctx, policiesPath, name, nil, nil, &p)
+	if err != nil || len(p.Policy) == 0 {
+		return nil, err
```

Comment:
In a scenario where err is nil and len(p.Policy)==0 , we will be sending nil, nil essentially.
May be we can consider creating an err and returning it.